### ISSUE TITLE
navigation is missing nested entrypoints

### ISSUE THREAD

**Issue by xixixao** [typedoc2md/typedoc-plugin-markdown#663]
### What package is the bug related to?

typedoc-plugin-markdown

### Describe the issue

With the configuration shown below, I only get the parent entry point in `MarkdownRendererEvent.navigation`, not the child one, and it also doesn't show up in children either.

![Screenshot 2024-08-01 at 16 15 20](https://github.com/user-attachments/assets/352414b0-35b8-41b2-b1a2-ec089f870630)

### TypeDoc configuration

```json
{
  "entryPoints": [
    "../src/nextjs/index.tsx",
    "../src/nextjs/server/index.tsx",
  ],
}
```

### Expected behavior

It should probably be included in children.

Btw the entry point config works fine and the page **is** generated, it's just not showing up in `MarkdownRendererEvent.navigation`.

**Comment by tgreyuk** [comment]
Thanks. Is you project on a public repo by an chance? (would be easier to debug).

**Comment by tgreyuk** [comment]
Should now be fixed in typedoc-plugin-markdown@4.2.6.

It is included as a child of nexjs:

<img width="430" alt="Screenshot 2024-08-25 at 10 00 48" src="https://github.com/user-attachments/assets/472ccf7e-e7dc-418d-a775-1af2bf663600">

<img width="779" alt="Screenshot 2024-08-25 at 09 59 34" src="https://github.com/user-attachments/assets/0dace4a2-3dd1-4033-aed2-49e7d42b5ca2">
